conf/config.toml

```diff
@@ -111,9 +111,9 @@
 
 [schedule]
 ## Controls the size limit of Region Merge.
-# max-merge-region-size = 54
+# max-merge-region-size = 20
 ## Specifies the upper limit of the Region Merge key.
-# max-merge-region-keys = 540000
+# max-merge-region-keys = 200000
```

Comment:
Do we also need to update doc in 8.3?

Reply:
All relative release notes have been reverted in doc and doc-cn.